src/modules/core/components/ProgressBar/ProgressBar.tsx

```diff
@@ -39,20 +41,35 @@ const ProgressBar = ({
   value = 0,
   max = 100,
   threshold,
+  hidePercentage = false,
 }: Props) => {
   const { formatMessage } = useIntl();
   const titleText = formatMessage(MSG.titleProgress, { value, max });
+  const visible = styles.thresholdVisibility;
+  const belowThreshold = styles.barColorBelowThreshold;
 
   return (
-    <div className={`${styles.wrapper} ${getMainClasses(appearance, styles)}`}>
+    <div
+      className={`${styles.wrapper} ${getMainClasses(
+        appearance,
+        styles,
+      )} ${classnames({ [belowThreshold]: threshold && value < threshold })}`}
+    >
       {!!threshold && (
         <div
           style={{
             left: `calc(${threshold}% - 14px)`,
           }}
           className={styles.threshold}
         >
-          <span className={styles.thresholdPercentage}>{threshold}%</span>
+          <span
+            className={classnames(styles.thresholdPercentage, {
+              [visible]: hidePercentage,
+            })}
+          >
+            {threshold}%
+          </span>
+
```

Comment:
Why is this needed in here?

Reply:
Are you referring to [visible]: hidePercentage or something else? If you are talking about the hide percentage, this is because for this progress bar we need to hide the threshold percentage but not remove it completely because it is used for the other progress bar showing the threshold percentage on top.